### ISSUE TITLE
Automatically update training data from source collector

### ISSUE THREAD

**Issue by josh-chamberlain** [Police-Data-Accessibility-Project/data-source-manager#89]
## Context

We're starting to generate more sources, but we don't update our training data.

## Requirements

- [x] We need a script which we can automate to update our training data—manually for now, just before we retrain (which is infrequent).
- [x] check for newly labeled stuff from our source collector app
   - [x] We want to grab not only items labeled as "relevant" and in our db, but also items labeled _not_ relevant.
- https://github.com/Police-Data-Accessibility-Project/data-source-identification/issues/324
- [ ] optional: apply [keyword extraction](https://github.com/Police-Data-Accessibility-Project/data-source-identification/pull/58#issuecomment-2123699871) to labeled data
- [x] #152 
- [x] update the hugging face training-urls dataset, with batch ID
  - #141 
  - place it there as raw data; we can transform it into more specific datasets as needed

**Comment by josh-chamberlain** [comment]
@maxachis I abstracted it—we don't use label studio, but our own labeling app should still cause better training data! we're a ways from having enough sources, though, so it's a `future` thing

**Comment by maxachis** [comment]
One wrinkle here is that we have multiple ways in which we can get data sources -- users or others can submit them into the data sources (DS) app, and our source collector (SC) can pick them up. The SC ones can get their HTML picked up right away, but the DS ones do not -- they would either need to be pulled into SC or pulled into a third repo which would replicate some of this, which is a little hinky.

One way we can help things along is to implement a `data sources sync` that's similar to our `agencies sync` logic that communicates Agencies between DS and SC, so that SC can have all of the URLs with the requisite data without and we can pull the HTML data and store it in a single location. That also helps separate the logic and have the SC be the more "backend" service, with DS being the more "frontend" service. 

Then, we can have SC add data to HuggingFace (HF), and DS doesn't have to concern itself with that.

**Comment by josh-chamberlain** [comment]
@maxachis yes, the timing is potentially kinky. Any time we do a periodic URL sweep, like for cleaning or metadata collection or archiving, we could add a step to get the HTML for any URLs for which it is missing.

**Comment by maxachis** [comment]
Note: Blocked by #338
